ecl/hqlcpp/hqlcpp.cpp

```diff
@@ -1226,6 +1234,8 @@ bool HqlCppInstance::useFunction(IHqlExpression * func)
     }
     if (include.length())
         useInclude(include.str());
+    if (source.length())
```

Comment:
This makes me wonder, would it be an idea to add isEmpty() { return !buffer||!*buffer; } to StringBuffer as there is for StrngAttr? I will open a jira as a suggestion.